contracts/context-config/src/repr.rs

```diff
@@ -0,0 +1,145 @@
+use std::{fmt, ops::Deref};
+
+use near_sdk::{bs58, near};
+use thiserror::Error;
+
+#[derive(Eq, Ord, Copy, Clone, Debug, PartialEq, PartialOrd)]
+#[near(serializers = [borsh, json])]
+#[serde(transparent)]
+pub struct Repr<T> {
```

Comment:
What does this mean? Is this "Representation"?

Reply:
`Repr` makes generic bytes representable over the wire, encoding them as `bs58`.